### ISSUE TITLE
Trying to retrieve all components of a Metadata Type via Org Browser fails

### ISSUE THREAD

**Issue by banderson5144** [forcedotcom/salesforcedx-vscode#2634]
### Summary

Whenever I try to retrieve all metadata of a specific type (I.e. Visualforce Components):

![image](https://user-images.githubusercontent.com/2395774/96154468-d68b9d00-0edc-11eb-9615-63cbea9ff5d6.png)

It throws an error:

```
Error running command sfdx.force.source.retrieve.component: data.forEach is not a function.
This is likely caused by the extension that contributes sfdx.force.source.retrieve.component.
```

### Steps To Reproduce:

1.  Try to download all metadata of a single type via Org Browser

### Expected result

Metadata should be downloaded

### Actual result

Throws error message described above

### Additional information

_Feel free to attach a screenshot_.

**VS Code Version**: 1.50.1

**SFDX CLI Version**: sfdx-cli/7.75.1 win32-x64 node-v12.18.4

**OS and version**: Windows 7 64-Bit

**Comment by usppat** [comment]
I'm also affected by that issue.
VS Code Version: 1.50.1
SFDX CLI Version: both 7.74.1 and 7.75.1
OS: Windows 10

**Comment by brosiusad** [comment]
I just got the same issue
VS Code Version 1.50.1
CLI: 7.75.0
OS: macOS 10.15.7

**Comment by lcampos** [comment]
Closing this since the fix was released in version 50.2.0 of the Salesforce extensions for VSCode. Thanks for reporting this.

**Comment by dkadam77** [comment]
This issue is again re-appeared in 50.8 ...can you pls fix it?
